cctools/as/arm.c

```diff
@@ -1448,7 +1448,7 @@ parse_reg_list (char ** strp)
 	      cur_reg = reg;
 	    }
 	  while (skip_past_comma (&str) != FAIL
-		 || (in_range = 1, *str++ == '-'));
+		 || ((void)(in_range = 1), *str++ == '-'));
```

Comment:
if this is just to silence `-Wcomma`, I generally think it's better to just ignore that warning, or to apply some other method of solving it, instead of applying its suggestion to add a `(void)` cast. But I guess that's a matter of opinion.